server/models/index.js

```diff
@@ -0,0 +1,54 @@
+//both: name, email, password, rating, city
+//customer: jobs: name, description, required skills, 
+//need date, city, job by id, proposals by id
+//contractor: skills, jobs completed.  Proposals: contractor
+//name, cost estimate, start date estimate, time frame estimate
+
+
+
+const baseOptions = {
+    discriminatorKey: 'userType',
+    collection: 'handymanDB',
+}
+
```

Comment:
On the userSchema we want to add a pre-save method to encrypt the password and a method to verify the correct password. Reference the User model in wk 22 activity 22 for example code. I think it would make sense to define the userSchema itself first, then the methods, then create the model and export it.